src/opensles/AudioStreamOpenSLES.cpp

```diff
@@ -66,9 +66,6 @@ SLuint32 AudioStreamOpenSLES::getDefaultByteOrder() {
 }
 
 Result AudioStreamOpenSLES::open() {
-
-    LOGI("AudioStreamOpenSLES::open() chans=%d, rate=%d", mChannelCount, mSampleRate);
```

Comment:
I really having a log where the parameters are logged somewhere. If this is not logged anywhere else, I prefer leaving this here.

Reply:
This only logs 2 out of a dozen parameters. Maybe we should log more.

This spews into the logs when I am testing.  Maybe:

    #ifndef OBOE_SUPPRESS_LOG_SPAM